etc/client.api.report.api.md

```diff
@@ -268,6 +268,15 @@ export interface DataValueWireToClient {
 // @public (undocumented)
 export type DefaultToFalse<B extends boolean | undefined> = false extends B ? false : undefined extends B ? false : true;
 
+// @public (undocumented)
+export type DisplayMetadata = {
+    displayName: string;
+    pluralDisplayName: string;
+    description?: string;
+    icon: Icon;
+    visibility: ObjectTypeVisibility;
```

Comment:
This is good but for the end game we might just want to add these two things to the `ObjectDefinition`